src/NZF1.jl

```diff
@@ -0,0 +1,29 @@
+# A. Montoison, Montreal, 04/2018.
+
+export NZF1
+
+function NZF1(n :: Int=26)
+  
+  mod(n,13) != 0 && error(": number of variables must divisible by 13")
```

Comment:
Instead of throwing an error, we prefer a warning, and we adjust `n` to the nearest multiple of 13.